Gems/Atom/Tools/AtomToolsFramework/Code/Source/Window/AtomToolsMainWindow.cpp

```diff
@@ -110,26 +100,20 @@ namespace AtomToolsFramework
         return names;
     }
 
-    void AtomToolsMainWindow::SetupMenu()
+    void AtomToolsMainWindow::CreateMenu()
     {
-        // Generating the main menu manually because it's easier and we will have some dynamic or data driven entries
-        m_menuFile = m_menuBar->addMenu("&File");
+        m_menuBar = new QMenuBar(this);
+        m_menuBar->setObjectName("MenuBar");
+        setMenuBar(m_menuBar);
     }
 
-    void AtomToolsMainWindow::SetupTabs()
+    void AtomToolsMainWindow::CreateTabBar()
     {
-        // The tab bar should only be visible if it has active documents
-        m_tabWidget->setVisible(false);
-        m_tabWidget->setTabBarAutoHide(false);
-        m_tabWidget->setMovable(true);
-        m_tabWidget->setTabsClosable(true);
-        m_tabWidget->setUsesScrollButtons(true);
-
-        // Add context menu for right-clicking on tabs
-        m_tabWidget->setContextMenuPolicy(Qt::ContextMenuPolicy::CustomContextMenu);
-        connect(m_tabWidget, &QWidget::customContextMenuRequested, this, [this]() {
-            OpenTabContextMenu();
-        });
+        m_centralWidget = new QWidget(this);
```

Comment:
I think the central widget needs to be created regardless of tabs and should go back to the constructor